index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bassem shams portofolio</title>
+    <link rel="stylesheet" href="styles.css" type="text/css"/>
+</head>
+<body>
+  <div class="contain">
+      <header class="upper">
+        <img class="image"  src="assests/images/ic_menu.png" alt="the menu icon"></header>
```

Comment:
[ Required Changes ]
- [ ] Please place the hamburger( ic_menu img) menu inside a `nav` tag